pkg/reconciler/labeler/v2/accessors.go

```diff
@@ -182,31 +190,41 @@ func NewConfigurationAccessor(
 }
 
 // list implements Accessor
-func (c *Configuration) list(ns, name string) ([]kmeta.Accessor, error) {
-	cl, err := c.configurationLister.Configurations(ns).List(labels.SelectorFromSet(labels.Set{
-		serving.RouteLabelKey: name,
-	}))
+func (c *Configuration) list(ns, routeName string, state v1.RoutingState) ([]kmeta.Accessor, error) {
+	cl, err := c.configurationLister.Configurations(ns).List(labels.Everything())
```

Comment:
I think this will blow up our scale tests since it'll be allocation all known configuration

Reply:
The expected case is zero, one, or few, so we can change the allocation of the copy array. That said is this List() call problematic? It was my understanding that these are pointers to the informerCache rather than a new allocation.

Reply:
> That said is this List() call problematic?

Yeah it creates a slice and appends to it as it filtering based on the label selector. With `labels.Everything()` you're creating a copy of what's the informer

Reply:
I've updated to directly apply the filter to the indexer